services/brig/src/Brig/API/User.hs

```diff
@@ -324,10 +324,12 @@ checkRestrictedUserCreation new = do
   when
     ( restrictPlease
         && or [isJust memail, isJust mphone]
-        && and [isNothing msso, isNothing mteam]
+        && and [isNothing msso, isNotTeamMember mteam]
```

Comment:
This implies that [the doc](https://github.com/wireapp/wire-server/blob/0e5bf97618cb4331f29c9005002f2d971fc64f4e/docs/reference/user/registration.md#details) added in the previous PR lies about email/phone being present as sufficient indicator of it being either a team creator or personal accounts. Let's also edit that.